src/core/Button/style.ts

```diff
@@ -1,23 +1,40 @@
 import styled from "@emotion/styled";
-import { Button } from "@material-ui/core";
+import { Button, ButtonProps } from "@material-ui/core";
 import {
   fontCapsXxxs,
+  getBorders,
   getColors,
   getCorners,
   getSpaces,
   Props,
 } from "../styles";
 
-const sdsPropNames = ["isAllCaps", "isRounded", "sdsStyle", "sdsType"];
+export interface ExtraProps {
+  color?: "primary" | "error";
+  isAllCaps?: boolean;
+  isRounded?: boolean;
+  sdsStyle?: "minimal" | "rounded" | "square";
+  sdsType?: "primary" | "secondary";
+}
+
+// Please keep this in sync with the props used in `ExtraProps`
+const doNotForwardProps = [
+  "isAllCaps",
+  "isRounded",
+  "sdsStyle",
+  "sdsType",
+  "color",
```

Comment:
ok, so I think not forwarding `color` to the MUI button is causing issues.  If you look at what was previously on line 11:

```
const sdsPropNames = ["isAllCaps", "isRounded", "sdsStyle", "sdsType"];
```

we did not have color so we did forward it.  I'm beginning to think that the code reuse we're trying to achieve here may not be sustainable in some ways.  I'll start a comment on the overall PR about that.